front/src/components/member/MemberUpdateModal.vue

```diff
@@ -35,12 +38,22 @@
 <script>
 import api from "@/api/member";
 
+const NICKNAME_REGEX = "^[a-zA-Zㄱ-ㅎㅏ-ㅣ가-힣0-9]+$";
+
 export default {
   name: "PostCreateModal",
   data() {
     return {
       rendered: false,
       newNickname: "",
+      rules: {
+        violated: (newVal) =>
+          !!newVal.match(NICKNAME_REGEX) ||
```

Comment:
```suggestion
          NICKNAME_REGEX.test(newVal) ||
```

느낌표 두개 안쓰는 방법~
(이거 하려면 REGEX는 아래로 수정되어야함)
`/^[a-zA-Zㄱ-ㅎㅏ-ㅣ가-힣0-9]+$/`